test/e2e/storage/testsuites/volume_expand.go

```diff
@@ -265,22 +265,26 @@ func ExpandPVCSize(origPVC *v1.PersistentVolumeClaim, size resource.Quantity, c
 	pvcName := origPVC.Name
 	updatedPVC := origPVC.DeepCopy()
 
+	// Retry the update on error, until we hit a timeout.
 	waitErr := wait.PollImmediate(resizePollInterval, 30*time.Second, func() (bool, error) {
 		var err error
 		updatedPVC, err = c.CoreV1().PersistentVolumeClaims(origPVC.Namespace).Get(pvcName, metav1.GetOptions{})
 		if err != nil {
-			return false, fmt.Errorf("error fetching pvc %q for resizing with %v", pvcName, err)
+			return false, fmt.Errorf("error fetching pvc %q for resizing: %v", pvcName, err)
 		}
 
 		updatedPVC.Spec.Resources.Requests[v1.ResourceStorage] = size
 		updatedPVC, err = c.CoreV1().PersistentVolumeClaims(origPVC.Namespace).Update(updatedPVC)
 		if err == nil {
 			return true, nil
 		}
-		framework.Logf("Error updating pvc %s with %v", pvcName, err)
+		framework.Logf("Error updating pvc %s: %v", pvcName, err)
```

Comment:
this can be saved into a larger scoped variable for return with the error (I think?)

Reply:
To be honest this seems like an inappropriate place for a retry loop. Why would update fail and then succeed, except for a version conflict?

Since I don't want to answer that question in this PR, I've taken your suggestion of providing better error context.